spec/index.bs

```diff
@@ -2183,20 +2183,20 @@ dictionary GPUBindGroupLayoutEntry {
     required GPUShaderStageFlags visibility;
     required GPUBindingType type;
 
-    // Used for uniform buffer and storage buffer bindings.
-    boolean hasDynamicOffset = false;
+    // Used for uniform buffer and storage buffer bindings. Must be undefined if not relevant.
```

Comment:
To be completely pedantic, if I understand correctly, only `?` things can be undefined.
Since this is not "?", it can be null or not null.

Reply:
Maybe add "?" for each of them? @kvark and @kainino0x

Reply:
Undefined is treated the same way as unset (i.e. `{}` and `{hasDynamicOffset: undefined}` are the same): 

> In the ECMAScript binding, a value of undefined for the property corresponding to a dictionary member is treated the same as omitting that property. Thus, it will cause an error if the member is required, or will trigger the default value if one is present, or will result in no entry existing in the dictionary value otherwise.

https://heycam.github.io/webidl/#idl-dictionaries